CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+## Contribution Guidelines
+
+- **All code PRs should**:
+  - have a meaningful commit message description
+  - comment important things
+  - include unit tests (positive and negative)
+  - pass [CI](http://drone.geirsson.com:8001/scalameta/scalameta)
+  - be reformatted with `./bin/scalafmt`
+  - add a relevant entry to the [changelog](https://github.com/scalameta/scalameta/tree/master/changelog) for the next stable release
+- **Be prepared to discuss/argue-for your changes if you want them merged**! You will probably need to refactor so your changes fit into the larger codebase
+- **If your code is hard to unit test, and you don't want to unit test it, that's ok**. But be prepared to argue why that's the case!
+- **It's entirely possible your changes won't be merged**, or will get ripped out later. This is also the case for my changes, as the Author!
```

Comment:
Perhaps the use of first person here is wrong.

How about "This is also the case for contributor changes, even `@xeno-by`!"